From 8cfc0cdbcd2931d335abefe93dc441527a008c3a Mon Sep 17 00:00:00 2001
From: Alek Du <alek.du@intel.com>
Date: Wed, 19 Dec 2018 13:14:12 +0000
Subject: [PATCH 2/2] makefile: install-debug

Signed-off-by: Alek Du <alek.du@intel.com>
---
 Makefile | 9 ++++++++-
 1 file changed, 7 insertions(+), 0 deletion(-)

diff --git a/Makefile b/Makefile
index 87eb0c93..010e1d74 100644
--- a/Makefile
+++ b/Makefile
@@ -90,6 +90,13 @@ endif
 sbl-hypervisor-install:
 	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT)-sbl BOARD=apl-mrb FIRMWARE=sbl RELEASE=$(RELEASE) install
 
+hypervisor-install-debug:
+	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) BOARD=$(BOARD) FIRMWARE=$(FIRMWARE) RELEASE=$(RELEASE)
+	make -C $(T)/efi-stub HV_OBJDIR=$(HV_OUT) EFI_OBJDIR=$(EFI_OUT) all install-debug
+
+sbl-hypervisor-install-debug:
+	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT)-sbl PLATFORM=sbl RELEASE=$(RELEASE) install-debug
+
 devicemodel-install:
 	make -C $(T)/devicemodel DM_OBJDIR=$(DM_OUT) install
 
