From 2fdd6d9a21299db3df6227a64ae8d1956fe8ae5f Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Tue, 9 Oct 2018 15:28:30 -0500
Subject: [PATCH] makefile: install-debug

---
 Makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile b/Makefile
index e45451d8..faf7a3c4 100644
--- a/Makefile
+++ b/Makefile
@@ -52,6 +52,12 @@ hypervisor-install:
 sbl-hypervisor-install:
 	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT)-sbl PLATFORM=sbl RELEASE=$(RELEASE) install
 
+hypervisor-install-debug:
+	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLATFORM) RELEASE=$(RELEASE) install-debug
+
+sbl-hypervisor-install-debug:
+	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT)-sbl PLATFORM=sbl RELEASE=$(RELEASE) install-debug
+
 devicemodel-install:
 	make -C $(T)/devicemodel DM_OBJDIR=$(DM_OUT) install
 
-- 
2.19.1

